dashboard/app/jobs/concerns/ai_rubric_config.rb

```diff
@@ -6,7 +6,7 @@ class AiRubricConfig
   #
```

Comment:
it looks like the reports in the new release directory may have been copied from the previous release and are out of date. can you please remove them? I assume this means the cached_responses are out of date too, in which case those would be great to remove as well.